### PR TITLE
fix security popup for first sfmi enable on macOS

### PR DESCRIPTION
**Pull request by songgao**
r @keybase/react-hackers / cc @keybase/hotpotatosquad

### CODE REVIEW

File: shared/fs/banner/system-file-manager-integration-banner/kext-permission-popup-container.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import {namedConnect, RouteProps} from '../../../util/container'
 import * as FsGen from '../../../actions/fs-gen'
 import KextPermissionPopup from './kext-permission-popup'
+import * as RouteTreeGen from '../../../actions/route-tree-gen'
 
 type OwnProps = RouteProps<{}, {}>
```
Comment (jzila) [inline shared/fs/banner/system-file-manager-integration-banner/kext-permission-popup-container.tsx]: You can get rid of `OwnProps` now, right?

**Comment by jzila** [comment]
That's fine, but at the least it's not longer a RouteProps, right?

**Comment by chrisnojima** [comment]
Correct. In the upcoming types connect pr we'll use ownprops (like before) so best to keep it named

**Comment by songgao** [comment]
It's from the router, so if typed I assume it'd still need to be RouteProps?

**Comment by songgao** [comment]
Anyway I'm merging this for now. It's an easy fix either way

**Pull request merged**